api_core/google/api_core/protobuf_helpers.py

```diff
@@ -36,3 +41,40 @@ def from_any_pb(pb_type, any_pb):
                 any_pb.__class__.__name__, pb_type.__name__))
 
     return msg
+
+
+def check_oneof(**kwargs):
+    """Raise ValueError if more than one keyword argument is not none.
+    Args:
+        kwargs (dict): The keyword arguments sent to the function.
+    Returns: None
+    Raises:
+        ValueError: If more than one entry in kwargs is not none.
+    """
+    # Sanity check: If no keyword arguments were sent, this is fine.
+    if not kwargs:
+        return None
```

Comment:
you can just do `return` instead of explicitly returning `None`, this is only useful if there is a real return value possible.

Reply:
Done.